extensions/systemd/systemd.js

```diff
@@ -60,12 +60,12 @@ class SystemdProcessManager extends cli.ProcessManager {
 
     isEnabled() {
         try {
-            execa.shellSync(`systemctl is-enabled ${this.systemdName}`);
+            execa.shellSync(`sudo systemctl is-enabled ${this.systemdName}`);
```

Comment:
Sudo commands should be run using `this.ui.sudo({{command_to_run}})`

Reply:
As much as it would be nice to do that here - the sudo helper returns a promise, and as such won't work without a decent amount of refactoring in other parts of the code 😕

Reply:
that's right, oops 🙊 

The very minimum is to add logging in that case, right? 🤔